src/odemis/odemisd/mdupdater.py

```diff
@@ -359,6 +362,20 @@ def updatePosition(unused, analyzer=analyzer, comp_affected=comp_affected):
 
         return True
 
+    def observeStreakLens(self, streak_lens, comp_affected):
+        """Update the magnification of the streak lens affecting the
+        streak readout camera."""
+
+        def updateMagnification(unused, streak_lens=streak_lens, comp_affected=comp_affected):
```

Comment:
Actually the `unused` variable is the magnification. So you could rename it `mag`, and avoid reading it again.

Reply:
Should we do the same then for the observePol* functions?

Reply:
Humm... indeed, good catch.
Could you have a look at this (in a separate PR)?